src/detectors/BEMC/Cluster_factory_EcalBarrelTruthClusters.h

```diff
@@ -40,7 +40,7 @@ class Cluster_factory_EcalBarrelTruthClusters : public JFactoryT<edm4eic::Cluste
         // for endcaps.
         m_enableEtaBounds=false;//{this, "enableEtaBounds", false};
 
-
+        app->SetDefaultParameter("BEMC:EcalBarrelTruthClusters:input_protoclust_tag", m_input_protoclust_tag, "Name of input collection to use");
```

Comment:
duplicated on line 48 though this has the description

Reply:
Fixing this too ...